src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/RectangleSelection.js

```diff
@@ -1,136 +1,77 @@
 // @flow
-import type {Node} from 'react';
-import {observer} from 'mobx-react';
-import {computed} from 'mobx';
 import React from 'react';
+import type {Node} from 'react';
 import withContainerSize from '../withContainerSize';
-import type {Normalizer, RectangleChange, SelectionData} from './types';
-import ModifiableRectangle from './ModifiableRectangle';
-import PositionNormalizer from './normalizers/PositionNormalizer';
-import RatioNormalizer from './normalizers/RatioNormalizer';
-import RoundingNormalizer from './normalizers/RoundingNormalizer';
-import SizeNormalizer from './normalizers/SizeNormalizer';
+import type {SelectionData} from './types';
+import RectangleSelectionRenderer from './RectangleSelectionRenderer';
 import rectangleSelectionStyles from './rectangleSelection.scss';
 
 type Props = {
+    backdrop: boolean,
     children?: Node,
-    containerHeight: number,
-    containerWidth: number,
+    disabled: boolean,
+    forceRatio: boolean,
```

Comment:
Previously, the `RatioNormalizer` was always used, if `minWidth` and `minHeight` both were set. But for the `ImageMap`, the `RatioNormalizer` shouldn't be used. Therefore i had to introduce this new prop. But since the default value is `true`, the behaviour shouldn't change

Reply:
With whom did you clarify that the behavior should be different in these two cases?

Reply:
With nobody, I used https://www.image-map.net/ as template

Reply:
The behaviour remains the same for existing usages of the RectangleSelection as well as for new usages, if you just omit the `forceRatio` prop. So why is that a problem?

Reply:
The problem for me is that the same component is behaving differently from a user perspective as well. If this is really the desired behavior I don't have a problem with that property. However, if the behavior should always be consistent, then I would prefer to not have this prop at all, to not even allow the behavior to diverge.